scripts/content-repo-preview/build.ts

```diff
@@ -0,0 +1,116 @@
+import { execFileSync } from 'child_process'
+import path from 'path'
+import fs from 'fs'
+
+function checkEnvVars() {
+  // Filter out defined env vars, leaving only the missing ones
+  const missingEnvVars = ['REPO', 'DEV_IO', 'IS_CONTENT_PREVIEW'].filter(
+    (key) => !process.env[key]
+  )
+
+  if (missingEnvVars.length > 0) {
+    console.error(
+      `Missing required environment variables: ${missingEnvVars.join(
+        ', '
+      )}. Ensure they're defined in .env or in the Vercel project`
+    )
+    return false
+  }
+
+  return true
+}
+
+async function main() {
+  if (!checkEnvVars()) {
+    process.exit(1)
+    return
+  }
```

Comment:
@zchsh  I've added the env var check here. I went with checking that all three are defined, `REPO`, `DEV_IO`, and `IS_CONTENT_PREVIEW`. If one/some of these aren't needed we can adjust later!